app/gui/src/ide/integration.rs

```diff
@@ -150,4 +189,23 @@ impl Integration {
             futures::future::ready(())
         }));
     }
+
+    fn update_welcome_screen(&self) {
```

Comment:
Maybe the name could be more specific. Like `update_projects_in_welcome_screen` or `refresh_welcome_Screen_project_list`.

Reply:
fixed